### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by atanasovRazvan**
Added the functionality and applied tests.
Home page content will update with job feed

### CODE REVIEW

File: src/home-page/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from 'react';
+import {
+  AppBar, Button, IconButton, Toolbar, Typography,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { makeStyles } from '@material-ui/styles';
+import { AuthContext } from '../context/AuthProvider';
+
+const useStyles = makeStyles((theme) => ({
```
Comment (ih90) [inline src/home-page/NavBar.js]: Now you have two different ways to apply CSS. Not great, not terrible.

File: src/home-page/__tests__/NavBar.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <NavBar />
+      </AuthContext.Provider>,
+    );
+  });
+
+  it('should display the user name', async () => {
+    expect(screen.getByText('Hello, razvan!')).toBeInTheDocument();
+  });
+
+  it('should setUsername to null if Logout button is pressed', async () => {
```
Comment (ih90) [inline src/home-page/__tests__/NavBar.test.js]: There is a mismatch between the description of the test and what it actually tests.

File: src/landing/LandingPage.js [hunk 3]
```diff
@@ -24,6 +29,8 @@ const LandingPage = () => {
     if (queryData && usernameState && passwordState) {
       if (queryData.users.find((user) => user.username === usernameState
           && user.password === passwordState)) {
+        setUsername(usernameState);
+        setUserRole(3);
```
Comment (ih90) [inline src/landing/LandingPage.js]: This seems wrong, user role needs to be the one the comes from the API. What if an admin or a company user logs in ?

File: src/utils/AppRouter.js [hunk 4]
```diff
@@ -2,14 +2,17 @@ import {
   BrowserRouter as Router, Redirect, Route, Switch,
 } from 'react-router-dom';
 import React from 'react';
-import LandingPage from './landing/LandingPage';
-import NotFoundPage from './NotFound';
+import NotFoundPage from '../notfound/NotFound';
+import PrivateRoute from './PrivateRoute';
+import HomePage from '../home-page/Home';
+import LandingPage from '../landing/LandingPage';
```
Comment (ih90) [inline src/utils/AppRouter.js:8]: Please pick one naming convention and try to stick with it. You have both camelCase and kebab-case for the directory names.

File: src/home-page/NavBar.js (hunk 1 above)
Comment (atanasovRazvan) [reply src/home-page/NavBar.js]: updated with styles.css file

File: src/home-page/__tests__/NavBar.test.js (hunk 2 above)
Comment (atanasovRazvan) [reply src/home-page/__tests__/NavBar.test.js]: yes, i actually forgot to change the name
fixed

File: src/landing/LandingPage.js (hunk 3 above)
Comment (atanasovRazvan) [reply src/landing/LandingPage.js]: fixed using the user.userRole there

File: src/utils/AppRouter.js (hunk 4 above)
Comment (atanasovRazvan) [reply src/utils/AppRouter.js:8]: fixed, all directories are now kebab-cased

**Review by ih90 — APPROVED** [review]

**Pull request merged**